samples/AzureContainerRegistryCoreReceiver/Controllers/AzureContainerRegistryController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.WebHooks;
+using Newtonsoft.Json.Linq;
+
+namespace AzureContainerRegistryCoreReceiver.Controllers
+{
+    public class AzureContainerRegistryController : ControllerBase
+    {
+        [AzureContainerRegistryWebHook(Id = "It")]
+        public IActionResult AzureContainerRegistryForIt(string @event, JObject data)
+        {
+            if (!ModelState.IsValid)
```

Comment:
This pattern isn't needed anymore. Application models handling the `[AzureContainerRegistryWebHook]` will add a `ModelStateInvalidFilter` for this action.